dipy/denoise/noise_estimate.py

```diff
@@ -1,9 +1,11 @@
 from __future__ import division, print_function
 
+import warnings
 import numpy as np
 
 from scipy.special import gammainccinv
 from scipy.ndimage.filters import convolve
+from scipy.stats import mode
```

Comment:
the mode import is not used anymore, so maybe some other random stuff got in again :/